### PR TITLE
Create code snippet for bigtable_get_family

### LINKED ISSUE #2296: Create code snippet for bigtable_get_family
Need to create code snippet for documentation purposes.

corresponding node.js snippet is available [here](https://github.com/googleapis/nodejs-bigtable/blob/ef790d498cecdd978b93d64cd1bb2de5dfca0eab/samples/document-snippets/family.js#L55)

Note: This issue/task is outcome of #2253

### PR DESCRIPTION
**Pull request by rsimion**
Fixes  #2296

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/2326)

### CODE REVIEW

File: google/cloud/bigtable/examples/data_snippets.cc [hunk 1]
```diff
@@ -464,6 +464,33 @@ void SampleRowsCollections(google::cloud::bigtable::Table table, int argc,
   (std::move(table));
 }
 
+void GetFamily(google::cloud::bigtable::Table table, int argc, char* argv[]) {
+  if (argc != 1) {
```
Comment (remyabel) [inline google/cloud/bigtable/examples/data_snippets.cc]: argc needs to be fixed.

File: google/cloud/bigtable/examples/data_snippets.cc [hunk 2]
```diff
@@ -464,6 +464,33 @@ void SampleRowsCollections(google::cloud::bigtable::Table table, int argc,
...
+    throw Usage{"get-family: <project-id> <instance-id> <table-id>"};
+  }
+
+  //! [bigtable_get_family]
```
Comment (remyabel) [inline google/cloud/bigtable/examples/data_snippets.cc]: Add region tag too? I think it needs to be `[START bigtable_get_family]`.

**Comment by rsimion** [comment]
@remyabel 
added the region tag.

I agree with the argc  but probably a new PR (@coryan ) to change for  all of the snippets. They all follow the same format.

**Comment by remyabel** [comment]
> I agree with the argc but probably a new PR (@coryan ) to change for all of the snippets. They all follow the same format.

Ah my mistake. I see that the args are already being consumed here:

https://github.com/googleapis/google-cloud-cpp/blob/ae78cc12e7cf9f378663098f824634de8f712c3a/google/cloud/bigtable/examples/data_snippets.cc#L546

**Pull request merged**